experiments/standalone/Vegetation/no_vegetation.jl

```diff
@@ -253,8 +253,8 @@ axislegend(ax)
 save(joinpath(savedir, "no_veg_state.png"), fig)
 fig2 = Figure()
 ax = Axis(fig2[1, 1], xlabel = "Time (days)", ylabel = "Energy Fluxes")
-lines!(ax, sol.t ./ 24 ./ 3600, SW_abs, label = "SW")
-lines!(ax, sol.t ./ 24 ./ 3600, LW_abs, label = "LW")
+lines!(ax, sol.t ./ 24 ./ 3600, SW_n, label = "SW")
+lines!(ax, sol.t ./ 24 ./ 3600, LW_n, label = "LW")
 lines!(ax, sol.t ./ 24 ./ 3600, SHF, label = "SHF")
 lines!(ax, sol.t ./ 24 ./ 3600, LHF, label = "LHF")
 lines!(ax, sol.t ./ 24 ./ 3600, RE, label = "RE")
```

Comment:
What is RE?

Reply:
Root extraction. In the soil model, we include the energy of the water as part of the soil energy budget. so if water leaves the soil due to root extraction, there is some (small - a few W/m^2) energy flux into the canopy also. We just include this in the canopy energy budget to conserve energy